x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/trial_license_complete_tier/notes.ts

```diff
@@ -70,5 +72,27 @@ export default function ({ getService }: FtrProviderContext) {
         expect(responseToTest.body.data!.persistNote.note.version).to.not.be.eql(version);
       });
     });
+
+    describe('get notes', () => {
+      beforeEach(async () => {
+        await deleteAllNotes(es);
+      });
+
+      it('should retrieve all notes that have been created by a specific user', async () => {
+        await createNote(supertest, 'elastic', 'note created by elastic user');
+        await createNote(supertest, 'user1', 'note created by user1 user');
+        await createNote(supertest, 'user2', 'note created by user2 user');
```

Comment:
Consider wrapping this in `await Promise.all(...)` to parallelize these requests